example/lib/payment_slection_page.dart

```diff
@@ -373,13 +384,20 @@ class _PaymentSelectionPageState extends State<PaymentSelectionPage> {
                     FocusScope.of(context).unfocus();
                     var options = {
                       'key': key,
-                      'amount': 100,
+                      'amount': '20000',
                       'currency': 'INR',
-                      'email': 'ramprasad179@gmail.com',
-                      'contact': '9663976539',
+                      'email': 'epay.bypass@motilaloswal.com',
```

Comment:
@ramprasadAnand can you please remove these details and have the payload as more generic?

Reply:
Done !